build_tools/benchmarks/common/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+# Tests
+################################################################################
+
+iree_py_test(
+  NAME
+    linux_device_utils_test
+  SRCS
+    "linux_device_utils_test.py"
+)
+
+iree_py_test(
+  NAME
+    common_arguments_test
+  SRCS
+    "common_arguments_test.py"
+)
+
+if(IREE_BUILD_TESTS)
+  # TODO(#8708): Temporary solution to fix python path for tests.
+  set_property(TEST "build_tools/benchmarks/common/linux_device_utils_test"
```

Comment:
Also use `${BENCHMARKS_TOOL_PYTHON_DIR}` here?

Reply:
BENCHMARKS_TOOL_PYTHON_DIR is the full path of benchmark directory. Getting the proper test name is a little bit complicated (needs package_name macro). As this is a temporary solution, I'd like to keep it simple : )